test/functorch/test_vmap.py

```diff
@@ -4541,6 +4550,9 @@ def op(t, ignored):
     @parametrize('batched_input', ["first", "last", "none"])
     @parametrize('dim', [2, 3])
     def test_feature_dropout(self, device, randomness, batched_input, dim):
+        if TEST_WITH_TORCHDYNAMO and torch.device(device).type == 'cuda' and randomness in ('different', 'same'):
```

Comment:
Need to investigate this one.

Reply:
Interesting, what does the errr look like?

Reply:
Er, they pass locally for me too

Reply:
Oh this is just nondeterminism in the test suite. There's no guarantee that the mean is actually between 0.25 and 0.75. On a bad seed the mean can get to 0.11, which sounds like what is happening here.

I'd just delete those two checks